dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -69,6 +69,10 @@ CONFIG_ATTRIBUTE_IGNORED=SESN0311W: The {0} configuration attribute was specifie
 CONFIG_ATTRIBUTE_IGNORED.explanation=The specified configuration is ignored as it is provided by the specified cacheManagerRef.
 CONFIG_ATTRIBUTE_IGNORED.useraction=No action is required.
 
+UPDATED_CONFIG_NOT_USED_AT_RESTORE=SESN0312W: The HTTP Session Cache configuration attribute {0} has changed when restoring the server process. If the value of the attribute changes when restoring the server process, the application will not use the updated value.
```

Comment:
```suggestion
UPDATED_CONFIG_NOT_USED_AT_RESTORE=SESN0312W: The {0} HTTP Session Cache configuration attribute changed while the server process was restoring. If the value of the attribute changes before the server process is restored, the application does not use the updated value.
```

Reply:
The wording is here is awkward because it makes it sound like the configuration is dynamically changing when restoring the server.  Somehow we need to make it clear that when the server process was checkpointed it had some value configured for this attribute.  Then before restoring the server process the configuration attribute was modified such that on restore that updated attribute value is observed.  Here is where we warn the user that we observed a configuration change from when the server process checkpointed to the restored server process and that we know we cannot honor that configuration change from checkpoint to restore.

Reply:
revised my suggestion